### PR TITLE
Test change

### PR DESCRIPTION
**Pull request by UPovilas**
Fixed integration test paragraph fetch;
Modified some unit tests :)
Added new unit test for GraphService
The test coverage should now be around 90%

### CODE REVIEW

File: Typeracer/Controllers/GraphController.cs
Comment (RytasV) [inline Typeracer/Controllers/GraphController.cs:1]: Graph does no longer have wpm and accuracy lines. Might be because of changes in this file. Not sure

**Review by RytasV — DISMISSED** [review]
Wpm and accuracy lines are gone. Refer to GraphController

**Review by RytasV — DISMISSED** [review]
Lots of good work done

**Review by RytasV — APPROVED** [review]
Lots of great work done

**Review by elvinukas — COMMENTED** [review]
👍

**Review by elvinukas — APPROVED** [review]

**Pull request merged**